java/client/src/main/java/glide/api/RedisClusterClient.java

```diff
@@ -0,0 +1,129 @@
+package glide.api;
+
+import static glide.api.RedisClient.buildChannelHandler;
+import static glide.api.RedisClient.buildCommandManager;
+import static glide.api.RedisClient.buildConnectionManager;
+
+import glide.api.commands.ClusterBaseCommands;
+import glide.api.commands.ClusterServerCommands;
+import glide.api.commands.Transaction;
+import glide.api.models.ClusterValue;
+import glide.api.models.commands.InfoOptions;
+import glide.api.models.configuration.RedisClusterClientConfiguration;
+import glide.api.models.configuration.Route;
+import glide.connectors.handlers.ChannelHandler;
+import glide.managers.CommandManager;
+import glide.managers.ConnectionManager;
+import glide.managers.models.Command;
+import java.util.Map;
+import java.util.Optional;
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * Async (non-blocking) client for Redis in Cluster mode. Use {@link #CreateClient} to request a
+ * client to Redis.
+ */
+public class RedisClusterClient extends BaseClient
+        implements ClusterBaseCommands, ClusterServerCommands {
+
+    protected RedisClusterClient(ConnectionManager connectionManager, CommandManager commandManager) {
+        super(connectionManager, commandManager);
+    }
+
+    /**
+     * Async request for an async (non-blocking) Redis client in Cluster mode.
+     *
+     * @param config Redis cluster client Configuration
+     * @return a Future to connect and return a RedisClusterClient
```

Comment:
Seems like we are capitalizing the beginning of the descriptions in other javadocs, can you adjust here and in the other javadocs in this PR?
```suggestion
     * @return A Future to connect and return a RedisClusterClient
```